### PR TITLE
Add Josuttis' Belle Views to C++ compiler

### PR DESCRIPTION
**Pull request by correaa**
https://github.com/josuttis/belleviews/tree/main/sources

### CODE REVIEW

File: etc/config/c++.amazon.properties [hunk 1]
```diff
@@ -1931,6 +1931,13 @@ libs.abseil.staticliblink=absl_bad_any_cast_impl:absl_failure_signal_handler:abs
 libs.abseil.versions.trunk.version=trunk
 libs.abseil.versions.trunk.path=/opt/compiler-explorer/libs/abseil
 
+libs.belleviews.name=Belle Views
+libs.belleviews.url=https://github.com/josuttis/belleviews
+libs.belleviews.versions=trunk
+libs.belleviews.description=A library of C++ views that just works for all basic use cases as expected.
+libs.belleviews.versions.trunk.version=trunk
+libs.belleviews.versions.trunk.path=/opt/compiler-explorer/libs/belleviews/sources
```
Comment (RubenRBS) [inline etc/config/c++.amazon.properties:1939]: I think the final path is
```suggestion
libs.belleviews.versions.trunk.path=/opt/compiler-explorer/libs/belleviews/trunk/sources
```

**Review by RubenRBS — APPROVED** [review]
Thanks!

**Review by RubenRBS — CHANGES_REQUESTED** [review]
Meant to request changes instead of accepting

**Comment by correaa** [comment]
> Meant to request changes instead of acepting

I agree with the change.

**Review by RubenRBS — APPROVED** [review]

**Comment by correaa** [comment]
thanks. good day today🇦🇷.

**Comment by jeremy-rifkin** [comment]
This is now live

**Comment by correaa** [comment]
And it works now: https://godbolt.org/z/dqboqEbfh.

@josuttis

**Pull request merged**